### ISSUE TITLE
Soap duplicates White Items

### ISSUE THREAD

**Issue by MuteTiefling** [MehVahdJukaar/Supplementaries#718]
supplementaries-1.19.2-2.3.20
Forge: 43.2.14

When Soap is placed in a crafting grid with 64 White Wool, it consumes 1 wool and provides 64

Before taking wool
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/9543430/719ac538-f586-41da-8728-57b2a06d0f44)
After taking wool
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/9543430/972d87b0-5491-4937-9a42-bc15a7ad053e)

**Comment by MuteTiefling** [comment]
Seeing the same behavior with White Sheetmetal from Immersive Engineering: 
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/9543430/f68f04c5-abd0-4ef8-adcc-6587e64ea7e4)

And White Concrete
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/9543430/8ac2a783-d753-4555-9170-0fb24be912db)


Seems to be anything where the 'cleansed' item has the word white in the ID?

**Comment by MuteTiefling** [comment]
Confirmed to be happening to all of these

- "hexerei:infused_fabric_block_dyed_white"
- "comforts:sleeping_bag_white"
- "pneumaticcraft:plastic_brick_white"
- "immersiveengineering:slab_sheetmetal_colored_white"
- "the_bumblezone:string_curtain_white"
- "hexerei:waxed_infused_fabric_carpet_dyed_white"
- "create:white_valve_handle"
- "nomadictents:white_shamiyana_wall"
- "hexerei:waxed_infused_fabric_block_dyed_white"
- "minecraft:white_wool"
- "pneumaticcraft:smooth_plastic_brick_white"
- "immersiveengineering:sheetmetal_colored_white"
- "pneumaticcraft:wall_lamp_white"
- "supplementaries:flag_white"
- "minecraft:white_carpet"
- "create:white_seat"
- "quark:white_stool"
- "pneumaticcraft:wall_lamp_inverted_white"
- "minecraft:white_banner"
- "minecraft:white_concrete_powder"
- "thermal:white_rockwool"
- "comforts:hammock_white"
- "minecraft:white_concrete"
- "hexerei:infused_fabric_carpet_dyed_white"
- "create:white_toolbox"

**Comment by MehVahdJukaar** [comment]
Yeah it's all the same recipe. Apparently is not a thing in 1.20, I'll backport

**Comment by MuteTiefling** [comment]
awesome, thanks!

**Comment by AgentStormy44** [comment]
Can confirm this is true in the latest Fabric version as well

**Comment by MehVahdJukaar** [comment]
should be fixed

**Comment by MuteTiefling** [comment]
Which version was this fixed in? Still having the issue in 
supplementaries-1.19.2-2.3.24

**Comment by MehVahdJukaar** [comment]
Hmm

**Comment by MehVahdJukaar** [comment]
should actually be fixed now, at least in 1.20

**Comment by Xelbayria** [comment]
does the issue still persist in 1.19.2?

**Comment by MuteTiefling** [comment]
Yes

**Comment by meeprdib** [comment]
Mod versions on 1.19.2 and below are not supported
